tool/tbot/main.go

```diff
@@ -144,6 +155,10 @@ func onWatch(botConfig *config.BotConfig) error {
 }
 
 func onStart(botConfig *config.BotConfig) error {
```

Comment:
Not for this PR, but we should consider refactoring the bulk of this logic outside of `package main` and test it.

Reply:
Yeah, there's a lot of cleanup and refactoring needed here. Hoping to prioritize that sooner rather than later :slightly_smiling_face: